packages/flutter_tools/test/general.shard/migrations/swift_package_manager_integration_migration_test.dart

```diff
@@ -671,6 +755,183 @@ void main() {
         expect(testLogger.traceText, contains('project.pbxproj already migrated. Skipping...'));
       });
 
+      testWithoutContext('skips Runner PBXNativeTarget migration if already migrated', () async {
```

Comment:
There was no test that verified that the migrator could skip the PBXNativeTargets